### PR TITLE
Bug fix + sync of airflow.template + airflow.yaml(not really needed)

### PR DESCRIPTION
**Pull request by lfreina**
I get the error message:
no matches for kind "Deployment" in version "extensions/v1beta1".

I modified the files so that it runs.

I also moved the namespace declaration outside of the airflow.yaml file.

### CODE REVIEW

**Comment by lfreina** [comment]
Also sorry for the spacing but my IntelliJ changes the spacing automatically. Feel free to change it back if you don't agree with that but it was too much work to do it manually this time.

**Comment by bhavaniravi** [comment]
Hey thanks for the PR, At this point I think it's a good idea to not check-in the build directory since it is generated and maybe add a `sample_build`

Can you revert the changes in the build directory alone. PR would look much cleaner

**Comment by lfreina** [comment]
Sure thing. Let me do that.

**Comment by lfreina** [comment]
So, that took some time.  Maybe this is better ^.^

File: scripts/kube/deploy.sh [hunk 1]
```diff
@@ -146,9 +146,11 @@ fi
 kubectl delete -f $DIRNAME/postgres.yaml
 kubectl delete -f $BUILD_DIRNAME/airflow.yaml
 kubectl delete -f $DIRNAME/secrets.yaml
+kubectl delete -f $DIRNAME/namespace.yaml
```
Comment (bhavaniravi) [inline scripts/kube/deploy.sh:149]: It's not advisable to delete namespace everytime we deploy. Some of the resources like PVC doesn't have to be removed inorder to redeploy an app

Comment (lfreina) [reply scripts/kube/deploy.sh]: Sure. It should be done now ^^

**Review by dariuszsq — APPROVED** [review]

**Comment by lfreina** [comment]
Thanks for the approval @dariuszsq :smile: I can't merge it though :wink:

**Comment by bhavaniravi** [comment]
@lfreina and @dariuszsq Thanks for your patience and for keeping this up to date.

**Pull request merged**